packages/calendar/src/util/date/translations.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  cs,
+  da,
+  de,
+  enGB,
+  et,
+  es,
+  fi,
+  fr,
+  it,
+  lt,
+  lv,
+  nl,
+  nb,
+  pl,
+  ru,
+  sv
+} from "date-fns/locale";
+
+const locales = {
+  cs,
+  da,
+  de,
+  enGB,
+  et,
+  es,
+  fi,
+  fr,
+  it,
+  lt,
+  lv,
+  nl,
+  nb,
+  pl,
+  ru,
+  sv
+};
+const weekName = {
```

Comment:
Type weekName as `Record<Language,string>`

Reply:
Nice catch, It actually caught an error for the Polish language